eslint.config.mjs

```diff
@@ -0,0 +1,90 @@
+import { defineConfig, globalIgnores } from "eslint/config";
+import path from "path";
+import { fileURLToPath } from "url";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import zustandRules from "eslint-plugin-zustand-rules";
+import tsParser from "@typescript-eslint/parser";
+import js from "@eslint/js";
+
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  {
+    env: {
+      node: true,
+    },
+
+    files: [
+      "/packages/arb-token-bridge-ui/src/**/*.js",
+      "/packages/arb-token-bridge-ui/src/**/*.ts",
+      "/packages/arb-token-bridge-ui/src/**/*.tsx",
+    ],
```

Comment:
What about file in `/packages/arb-token-bridge-ui/src`? config file, synpress setup etc?